privacyidea/static/components/token/views/token.enrolled.push.html

```diff
@@ -9,10 +10,11 @@
             Please scan the QR code with the privacyIDEA Authenticator App
         </p>
         <img width="{{ qrCodeWidth }}"
+             ng-class="$state.includes('token.wizard') == true ? 'centered' : 'not-centered'"
              ng-src="{{ enrolledToken.pushurl.img }}"/>
     </div>
 
-    <div class="col-sm-4 pull-right" ng-show="enrolledToken.rollout_state === 'clientwait'">
+    <div class="col-sm-6" ng-show="enrolledToken.rollout_state === 'clientwait'">
```

Comment:
Why is it necessary to increase the image size from 4/12 to 6/12.
The 2 step enrollment has a lot on the right side, why make it smaller?

Reply:
The image was already at 6, I only increased the 2 step text width also to 6 to fill the box.